src/update-pagination.ts

```diff
@@ -38,77 +41,77 @@ export const content: Section[] = [
     title: 'sidebar.code_contribution',
     contents: [
       {
-        href: '/how-to-setup-freecodecamp-locally',
+        href: 'how-to-setup-freecodecamp-locally/',
```

Comment:
```suggestion
        href: '/how-to-setup-freecodecamp-locally/',
```
Otherwise the link will be messed up - `/{current page}/codebase-best-practices/`.

Ie. on `/how-to-proofread-files/` link will be `/how-to-proofread-files/how-to-setup-freecodecamp-locally/`

Reply:
Okay it should be fixed now.